tests/test-option-parser.sh

```diff
@@ -32,7 +32,7 @@ check_output_inner ()
     func_quote pretty ${1+"$@"}
     $ECHO "[[ output check ]] args: $func_quote_result"
 
-    output=`$helper ${1+"$@"} 2>/dev/null`
+    output=`$__GL_ALL_SHELLS_SHELL "$helper" ${1+"$@"} 2>/dev/null`
```

Comment:
On 2016-08-17 14:38, Pavel Raiskup wrote: 

> Thanks for your work! 
> 
> Maybe it is detail, but could we have rather $helper properly quoted and later eval()ed to not copy the ugly $__GL_ALL_SHELLS_SHELL on multiple places?

Sure, I actually did that initially, but then wondered if it might
affect the quote nesting.  I'll go back and test it with eval and append
the patch. 

Thanks, 

Scott